src/main/java/org/opensearch/ad/rest/handler/IndexAnomalyDetectorJobActionHandler.java

```diff
@@ -1,361 +1,361 @@
-                                    new StopDetectorRequest(detectorId),
-                                    stopAdDetectorListener(detectorId)
-                                )
-                        );
-                    }
-                } catch (IOException e) {
-                    String message = "Failed to parse anomaly detector job " + detectorId;
-                    logger.error(message, e);
-                    listener.onFailure(new OpenSearchStatusException(message, RestStatus.INTERNAL_SERVER_ERROR));
-                }
-            } else {
-                listener.onFailure(new OpenSearchStatusException("Anomaly detector job not exist: " + detectorId, RestStatus.BAD_REQUEST));
-            }
-        }, exception -> listener.onFailure(exception)));
-    }
-
-    private ActionListener<StopDetectorResponse> stopAdDetectorListener(String detectorId) {
-        return new ActionListener<StopDetectorResponse>() {
-            @Override
-            public void onResponse(StopDetectorResponse stopDetectorResponse) {
-                if (stopDetectorResponse.success()) {
-                    logger.info("AD model deleted successfully for detector {}", detectorId);
-                    // StopDetectorTransportAction will send out DeleteModelAction which will clear all realtime cache.
-                    // Pass null transport service to method "stopLatestRealtimeTask" to not re-clear coordinating node cache.
-                    adTaskManager.stopLatestRealtimeTask(detectorId, ADTaskState.STOPPED, null, null, listener);
-                } else {
-                    logger.error("Failed to delete AD model for detector {}", detectorId);
-                    // If failed to clear all realtime cache, will try to re-clear coordinating node cache.
-                    adTaskManager
-                        .stopLatestRealtimeTask(
-                            detectorId,
-                            ADTaskState.FAILED,
-                            new OpenSearchStatusException("Failed to delete AD model", RestStatus.INTERNAL_SERVER_ERROR),
-                            transportService,
-                            listener
-                        );
-                }
-            }
-
-            @Override
-            public void onFailure(Exception e) {
-                logger.error("Failed to delete AD model for detector " + detectorId, e);
-                // If failed to clear all realtime cache, will try to re-clear coordinating node cache.
-                adTaskManager
-                    .stopLatestRealtimeTask(
-                        detectorId,
-                        ADTaskState.FAILED,
-                        new OpenSearchStatusException("Failed to execute stop detector action", RestStatus.INTERNAL_SERVER_ERROR),
-                        transportService,
-                        listener
-                    );
-            }
-        };
-    }
-
-}
+/// *
```

Comment:
1. No, the build passed
2. ^^
3. The reason I didn't delete it because we will rebase it with the latest main later and these files will be needed. Even, I thought of having a different name for these files but since they import dependencies from Job Scheduler it fails during compile.